spec/spec_helper.rb

```diff
@@ -2,16 +2,17 @@
 require "capybara/rspec"
 require "factory_girl_rails"
 
-WebMock.disable_net_connect!(allow_localhost: true)
-
-# The `.rspec` file also contains a few flags that are not defaults but that
-# users commonly want.
-# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+WebMock.disable_net_connect!(allow: [/pusher/], allow_localhost: true)
 
 Capybara.ignore_hidden_elements = false
 Capybara.javascript_driver = :poltergeist
 Capybara.register_driver(:poltergeist) do |app|
-  Capybara::Poltergeist::Driver.new(app)
+  Capybara::Poltergeist::Driver.new(app,
+                                    js_errors: false,
+                                    phantomjs_options: [
+                                      "--ignore-ssl-errors=yes",
+                                      "--ssl-protocol=any"
+                                      ])
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.